app/helpers/notifications_helper.rb

```diff
@@ -4,40 +4,44 @@ module NotificationsHelper
 
   def object_link(note, actors)
     target_type = note.popup_translation_key
-    actors_count = note.actors.size
+    opts = {actors: actors, count: note.actors.size}
```

Comment:
Oh yeah, changing the locales is not what I intended. Your current change is OK, thanks 👍